x-pack/test_serverless/functional/test_suites/security/navigation.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getPageObject, getService }: FtrProviderContext) {
+  const svlSecLandingPage = getPageObject('svlSecLandingPage');
+  const svlSecNavigation = getService('svlSecNavigation');
+  const svlCommonNavigation = getPageObject('svlCommonNavigation');
+  const testSubjects = getService('testSubjects');
+  const browser = getService('browser');
+
+  describe('navigation', function () {
+    before(async () => {
+      await svlSecNavigation.navigateToLandingPage();
+    });
+
+    it('has security serverless side nav', async () => {
+      await svlSecLandingPage.assertSvlSecSideNavExists();
+      await svlCommonNavigation.expectExists();
+    });
+
+    it('breadcrumbs reflect navigation state', async () => {
+      await svlCommonNavigation.breadcrumbs.expectExists();
+      // TODO: use `deepLinkId` instead of `text`, once security deep links are available in @kbn/core-chrome-browser
```

Comment:
We are already setting the `navigationTree`, should we do something else?

Reply:
This isn't required, but nice to have. For example, observability and search have their deep link ids types in a separate package so that we can use it for type safety and auto complete in different places. 

https://github.com/elastic/kibana/blob/47dc8f5e7ce8576ebad1d197a82cfda789a0b724/packages/core/chrome/core-chrome-browser/src/project_navigation.ts#L39-L45